realm/realm-library/src/androidTest/java/io/realm/rule/TestRealmConfigurationFactory.java

```diff
@@ -109,4 +116,20 @@ public RealmConfiguration createConfiguration(String name, byte[] key) {
     public RealmConfiguration.Builder createConfigurationBuilder() {
         return new RealmConfiguration.Builder(getRoot());
     }
+
+    // Copies a Realm file from assets to temp dir
+    public void copyRealmFromAssets(Context context, String realmPath, String newName)
```

Comment:
The one in TestHelper is being used IOSRealmTest. No need to bother mark it as deprecated. I will just convert that on later today, just a few lines of code.